cts/cts-exec.in

```diff
@@ -1238,6 +1248,16 @@ def main(argv):
         show_usage()
         sys.exit(CrmExit.OK)
 
+    if opts.options['pacemaker-remote']:
+        daemon_name = "pacemaker-remoted"
+    else:
+        daemon_name = "pacemaker-execd"
+
+    if is_proc_running(daemon_name):
+        print(f"Error: {daemon_name} is already running!")
+        print(f"Run {sys.argv[0]} only when the cluster is stopped.")
```

Comment:
Pacemaker supports Python >=3.4, and sadly f strings were not introduced until 3.6

Reply:
Darn, will fix.